lib/srv/db/cloud/mocks.go

```diff
@@ -133,6 +136,73 @@ func (m *RDSMock) ModifyDBClusterWithContext(ctx aws.Context, input *rds.ModifyD
 	}
 	return nil, trace.NotFound("cluster %v not found", aws.StringValue(input.DBClusterIdentifier))
 }
+func (m *RDSMock) DescribeDBProxiesWithContext(ctx aws.Context, input *rds.DescribeDBProxiesInput, options ...request.Option) (*rds.DescribeDBProxiesOutput, error) {
+	if aws.StringValue(input.DBProxyName) == "" {
+		return &rds.DescribeDBProxiesOutput{
+			DBProxies: m.DBProxies,
+		}, nil
+	}
+	for _, dbProxy := range m.DBProxies {
+		if aws.StringValue(dbProxy.DBProxyName) == aws.StringValue(input.DBProxyName) {
+			return &rds.DescribeDBProxiesOutput{
+				DBProxies: []*rds.DBProxy{dbProxy},
+			}, nil
+		}
+	}
+	return nil, trace.NotFound("proxy %v not found", aws.StringValue(input.DBProxyName))
+}
+func (m *RDSMock) DescribeDBProxyEndpointsWithContext(ctx aws.Context, input *rds.DescribeDBProxyEndpointsInput, options ...request.Option) (*rds.DescribeDBProxyEndpointsOutput, error) {
+	inputProxyName := aws.StringValue(input.DBProxyName)
+	inputProxyEndpointName := aws.StringValue(input.DBProxyEndpointName)
+
+	if inputProxyName == "" && inputProxyEndpointName == "" {
+		return &rds.DescribeDBProxyEndpointsOutput{
+			DBProxyEndpoints: m.DBProxyEndpoints,
+		}, nil
+	}
+
+	endpoints := []*rds.DBProxyEndpoint{}
```

Comment:
nit: 
https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

```suggestion
	var endpoints  []*rds.DBProxyEndpoint
```